docs/extensions/readme.md

```diff
@@ -1173,3 +1175,56 @@ An object with an optional property.
   }
 }
 ```
+
+## x-ms-header-collection-prefix
+Use `x-ms-header-collection-prefix` to deal with prefixes for headers. Has different behavior if it refers to a request header or a response header:
+
+- Request header: All keys in the request headers will be prefixed with the prefix value before being sent to the service.
+- Response header: Only response headers that start with the prefix specified here will be returned to users. Additionally, the prefix will be stripped from the response header key before being returned to users
+
+Additionally, applying this extension to a schema forces the schema to become a dictionary.
+
+**Schema**: `string`. Name of the prefix you want to append / filter by. A common value for storage libraries is `x-ms-meta-`.
+
+**Request Example**:
+
+```json
+"parameters": [
+  "x-ms-meta": {
```

Comment:
nit: needs an object definition to be a valid JSON

Reply:
totally defied this one wrong, fixed it. thanks!